### PR TITLE
Benchmark time to call sendtoaddress with many UTXOs

### PR DESCRIPTION
**Pull request by str4d**
Part of #2156.

### CODE REVIEW

File: src/zcbenchmarks.cpp [hunk 1]
```diff
@@ -17,6 +17,7 @@
 #include "main.h"
 #include "miner.h"
 #include "pow.h"
+#include "rpcserver.h"
```
Comment (nathan-at-least) [inline src/zcbenchmarks.cpp]: Why is this change necessary?

File: qa/zcash/create_wallet_200k_utxos.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            taddr = self.nodes[0].getnewaddress()
+            inputs.append(self.nodes[1].sendtoaddress(taddr, Decimal("0.001")))
+            if i % 1000 == 0:
+                self.nodes[1].generate(1)
+                self.sync_all()
+
+        assert_equal(set(self.nodes[1].getrawmempool()), set(inputs))
+        print(self.nodes[1].getmempoolinfo())
+        self.nodes[1].generate(1)
+        print(self.nodes[1].getmempoolinfo())
```
[the remark this answers is not in the document]
Comment (str4d) [reply qa/zcash/create_wallet_200k_utxos.py]: This file is not an actual test; it was just the file I used to generate the benchmark dataset. As such, these values were in here as part of my checking things. I can remove them if desired.

File: qa/zcash/create_wallet_200k_utxos.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+        inputs = []
+        for i in range(200000):
+            taddr = self.nodes[0].getnewaddress()
+            inputs.append(self.nodes[1].sendtoaddress(taddr, Decimal("0.001")))
+            if i % 1000 == 0:
+                self.nodes[1].generate(1)
+                self.sync_all()
+
+        assert_equal(set(self.nodes[1].getrawmempool()), set(inputs))
```
[the remark this answers is not in the document]
Comment (str4d) [reply qa/zcash/create_wallet_200k_utxos.py]: Yeah, this is a buggy line that was left over from a previous version. I'll remove it and the print lines below; they are unnecessary.

File: src/zcbenchmarks.cpp (hunk 1 above)
Comment (str4d) [reply src/zcbenchmarks.cpp]: `ValueFromAmount()` is defined there.

File: qa/zcash/create_wallet_200k_utxos.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#
+# Create a large wallet
+#
+# To use:
+# - Copy to qa/rpc-tests/wallet_large.py
+# - Add wallet_large.py to RPC tests list
+# - ./qa/pull-tester/rpc-tests.sh wallet_large --noshutdown
```
Comment (bitcartel) [inline qa/zcash/create_wallet_200k_utxos.py]: --nocleanup rather than --noshutdown ?

Comment (str4d) [reply qa/zcash/create_wallet_200k_utxos.py]: I used `--noshutdown` because I wanted to double-check things afterwards, but yes it could be that for general usage.

**Comment by str4d** [comment]
All comments addressed. I will merge this once the CI server has been updated to run the benchmarks.

**Comment by str4d** [comment]
CI server has been updated.

@zkbot r+
